versioned/mongodb/src/main/java/com/dremio/nessie/versioned/store/mongodb/MongoDBStore.java

```diff
@@ -371,8 +370,8 @@ void resetCollections() {
 
   private Bson toBson(ConditionExpression conditionUnAliased) {
     final MongoDBAliasCollectorImpl collector = new MongoDBAliasCollectorImpl();
-    final ConditionExpressionAliasVisitor conditionExpressionAliasVisitor = new MongoDBConditionExpressionAliasVisitor();
-    final ConditionExpression aliased = conditionUnAliased.accept(conditionExpressionAliasVisitor, collector);
+    final MongoDBConditionAliasVisitor conditionAliasVisitor = new MongoDBConditionAliasVisitor();
+    final ConditionExpression aliased = conditionUnAliased.accept(conditionAliasVisitor, collector);
```

Comment:
I'm wondering if you need to pass the collector, or is it always paired with the alias visitor? If so, should it be part of the alias visitor?

Reply:
This was mimicking the Dynamo alias methods, but yes it is paired with the AliasVisitor. No need to expose it at all as far as I can see.

Reply:
Removed. Now instantiated in the AliasVisitor.